src/Listener/DoctrineHealthCheckListener.php

```diff
@@ -41,7 +41,8 @@ public function onLiveCheck (HealthCheckLiveEvent $event) : void
 			return;
 		}
 
-		$databaseUpdateSchemaList = $validator->getUpdateSchemaList();
+		// We'll ignore "DROP TABLE doctrine_migration_versions" since it shouldn't appear here
+		$databaseUpdateSchemaList = array_diff($validator->getUpdateSchemaList(), ["DROP TABLE doctrine_migration_versions"]);
```

Comment:
Can we rather filter out all entries containing `doctrine_migration_versions`? That would make it a bit more robust.